api/src/test/java/org/openmrs/module/patientflags/FlagServiceTest.java

```diff
@@ -1,17 +1,25 @@
 package org.openmrs.module.patientflags;
 
-import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 
-import junit.framework.Assert;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
 
+import org.junit.Before;
 import org.junit.Test;
+import org.openmrs.Cohort;
 import org.openmrs.Patient;
+import org.openmrs.api.PatientService;
 import org.openmrs.api.context.Context;
 import org.openmrs.module.patientflags.Flag;

```

Comment:
These are shown as unused imports, do we need these?

Reply:
They will use when we writing test for tags and priority services.

Reply:
There's no need to keep imports for future use. If they aren't currently being used, you could remove them.